### PR TITLE
Implement new blast settings combination for gap-open/close and match/mismatch scores

### PR DESCRIPTION
**Pull request by imransl**
## Description
The default values and behaviour of some BLAST parameters such as gap penalties, match/mismatch and matrix scores have been updated to reflect the BLAST in the current site.

### TODO:
- [ ] Check and update the default values for all sensitivity presets (gap penalties are especially wrong; but don't trust anything else either)
- [x] Remove from sensitivity presets the settings that are irrelevant for a program (gap penalties, gap alignment)
- [x] Remove the `NONE` option for the scoring matrix, and update the tests
- [x] Update the UI to match the design
- [x] Disable the gap align checkbox

## Related JIRA Issue(s)
[ENSWBSITES-1838](https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-1838)

## Deployment URL(s)
http://blast-default-values.review.ensembl.org

## Views affected
BLAST

### CODE REVIEW

File: src/content/app/tools/blast/state/blast-api/blastApiSlice.ts [hunk 1]
```diff
@@ -66,9 +67,14 @@ export type RejectedJob = {
...
-        url: `${config.toolsApiBaseUrl}/blast/config`
-      }),
+      queryFn: () => {
+        return Promise.resolve({
+          data: mockConfig as unknown as BlastSettingsConfig
+        });
+      },
+      // query: () => ({
+      //   url: `${config.toolsApiBaseUrl}/blast/config`
+      // }),
```
Comment (azangru) [inline src/content/app/tools/blast/state/blast-api/blastApiSlice.ts:77]: **REMINDER TO US ALL:** remember to change this back after the backend has updated the config

File: src/content/app/tools/blast/components/blast-settings/BlastSettings.tsx [hunk 2]
```diff
@@ -85,6 +85,23 @@ const BlastSettings = ({ config }: Props) => {
...
+  /**
+   * - change select element for gap open and gap extend
+   * - update blastFormSlice
+   *  - when updating the program, need to make sure match/mismatch or matrix
+   *    get updated accordingly (and gap open/gap extend too)
+   *  - need to introduce a function for updating match/mismatch and matrix
+   *    (maybe they be part of the function for updating a single parameter)
+   * - update CSS of the parameters section
+   */
+
```
Comment (jyothishnt) [inline src/content/app/tools/blast/components/blast-settings/BlastSettings.tsx:104]: remove?

**Comment by jyothishnt** [comment]
Gap penalties width needs to be longer to accommodate bigger numbers
Screenshot 2023-02-15 at 16 34 23

File: src/content/app/tools/blast/components/blast-settings/BlastSettings.tsx [hunk 3]
```diff
@@ -270,7 +299,15 @@ const BlastSettings = ({ config }: Props) => {
               onChange={(value: string) =>
                 onBlastParameterChange('gapext', value)
               }
```
[the remark this answers is not in the document]
Comment (imransl) [reply src/content/app/tools/blast/components/blast-settings/BlastSettings.tsx]: ✅

**Pull request merged**